src/GoogleSignin.js

```diff
@@ -41,9 +41,9 @@ class GoogleSignin {
     this.configPromise = RNGoogleSignin.configure(options);
   }
 
-  async signInSilently() {
+  async restorePreviousSignIn() {
```

Comment:
please keep the previous code here, thanks

Reply:
resolved in 6b1632ef